config/webpack/webpack.config.js

```
@@ -1,17 +1,6 @@
- // name this file config/webpack/webpack.config.js
- const { env, webpackConfig } = require('shakapacker')
- const { existsSync } = require('fs')
- const { resolve } = require('path')
+// See the shakacode/shakapacker README and docs directory for advice on customizing your webpackConfig.
+const { generateWebpackConfig } = require('shakapacker')
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)